src/components/oscal-utils/OSCALRestUtils.tsx

```diff
@@ -241,12 +259,12 @@ export function createOrUpdateSspControlImplementationImplementedRequirementStat
   if (implementationSetParameters?.length) {
     statementByComponent["set-parameters"] ??= [];
```

Comment:
On lines: `239`, `253`, & `260` there is the error:
```
/OSCAL/oscal-react-library/src/components/oscal-utils/OSCALRestUtils.tsx(239,52): syntax error TS1109: Expression expected.
```
Despite both babel & typescript supporting Logical nullish assignment in their current versions.